examples/Kaleidoscope/utilities.jl

```diff
@@ -0,0 +1,11 @@
+# Move to LLVM.jl
+
+function position_before_terminator!(builder::LLVM.Builder, bb::LLVM.BasicBlock)
+    v = LLVM.API.LLVMGetBasicBlockTerminator(LLVM.blockref(bb))
```

Comment:
https://github.com/maleadt/LLVM.jl/blob/fbf6a393d25bd105d0ab8668d62425d4f3c2ca92/src/core/basicblock.jl#L33

Reply:
This throws if there is no terminator int he block.

Reply:
or given that it should be reasonably fast on 0.7 just return nothing in cases of a null pointer (out of scope for this PR, of course)...